lms/services/grading_info.py

```diff
@@ -43,27 +43,33 @@ def get_by_assignment(self, oauth_consumer_key, context_id, resource_link_id):
         :param resource_link_id: the assignment's resource_link_id
             (identifies the assignment within the LMS course)
         :type resource_link_id: str
-
-        :return: a list of all the (GradingInfo, HUser) tuples for the assignment
-        :rtype: list[(GradingInfo, HUser)]
```

Comment:
I removed the return type from the docstring. It's just described as a "list of student dicts" higher up in the docstring. I think it'd be silly to document the exact contents of these dicts. Just read the code. It should probably be replaced with a `StudentInfo` value rather than dicts then we wouldn't have this problem, but I'm not going that far here